game.py

```diff
@@ -0,0 +1,126 @@
+from pieces import *
+from board import *
+from ascii import *
+from os import system, name
+import time
+
+#need to fix knight moves and make it so it doesn't change players if an invalid move for a piece is picked
+
+def clear():
+    # for windows
+    if name == 'nt':
+      system('cls')
+    # for mac and linux(here, os.name is 'posix')
+    else:
+      system('clear')
+
+def turn(board, last_move):
+  while True:
+    clear()
+    if last_move is not None: print(last_move)
+    print("\n\n")
+    board.update_moves()
+    if board.white_to_move == True:
+      print("White's turn to move.\n\n")
+      current_player = "White"
+      opposing_player = "Black"
+    else:
+      print("Black's turn to move.\n\n")
+      current_player = "Black"
+      opposing_player = "White"
+    board.ascii.draw_board(board)
+    print("Enter ? for a list of commands or enter your move (e.g. 'move e7 to e5' or 'move g8 to f6').")
+    user_input = input()
+    if user_input == "?":
+      print("List of commands\n\n'?' for a list of commands\n'q' quits to the menu\n'draw' to request a draw\n'resign' to resign\n'save' to save the current state of the board\n'undo' to request an undo of the last move\n'board' to view the board\n'move' (e.g. 'move e7 to e5')\n")
+    elif user_input == "q":
+      return "break"
+    elif user_input == "resign":
+      while True:
+        print("Are you sure you want to resign? (y/n)")
+        resignation = input()
+        if resignation == "y":
+          print(current_player + " has resigned. " + opposing_player + " wins!")
+          time.sleep(3)
+          return "break"
+        if resignation == "n":
+          pass
+    elif user_input == "draw":
+      pass
```

Comment:
As part of merging this PR that now knows how to draw the board, let's fill in this implementation instead of `pass`ing.

Reply:
Got it. Will do.